src/encoded/static/scss/encoded/modules/_facet.scss

```diff
@@ -59,34 +60,20 @@
     padding-right: 0;
 }
 
-.clear-filters-control {
-    margin-bottom: 5px;
-    text-align: right;
-    font-size: 0.9rem;
-
-    @at-root #{&}--summary {
-        margin-top: -30px;
-        margin-bottom: 30px;
-        position: absolute;
-        bottom: 10px;
-        left: 0px;
-
-        @media print {
-            display: none;
-        }
-    }
-}
-
 .search-header-control {
     position: relative;
-    padding-bottom: 25px;
+    padding-bottom: 10px;
 }
 
-.col-sm-5 {
-    .clear-filters-control {
+.clear-filters-control {
+    position: relative;
+    height: 1.1rem;
+
```

Comment:
I added it. Yeah, that looked kind of tight. It makes the one on the Summary page look a tad far down, but we might want to address that in a future summary-page update.

I also got rid of the following .clear-filters-control-matrix definition because I don’t use it in matrix.js anymore.